### ISSUE TITLE
Journal no longer opens to current instance

### ISSUE THREAD

**Issue by benhamlin** [Meivyn/AdventureGuideLockouts#5]
When this addon is enabled, opening the dungeon journal while in a specific dungeon no longer opens that dungeon's panel. Instead, the top level 'dungeons' tab is shown. 

Minor issue, but would be nice to fix if easy. Thank you for this wonderful addon.

**Comment by Meivyn** [comment]
I don't have this issue. Do you get any error? Just tested with The motherlode, no problem on my side.

**Comment by Meivyn** [comment]
I can't reproduce your issue. Can you give me more details?

**Comment by benhamlin** [comment]
It looks like there's a little more to the repro. I disabled all my other addons to make sure nothing else was interfering, here's what I saw:

After zoning into Siege, when I bring up the dungeon journal it correctly opens directly to the Siege panel. However, after doing a /reload, it opens to Siege for a split-second, then switches to the Dungeons panel. I believe something else triggers it besides /reload since I see this all the time during regular usage, but not sure what. 

- Rezoning doesn't fix it.
- The first time after doing /reload, it lingers on the Siege panel for a second before it switches to the Dungeon panel. You can do this over and over and it takes longer to switch the first time after each /reload.
- None of this behavior repros without the addon enabled.

**Comment by Meivyn** [comment]
Can you please try replacing this (at the end of ``EncounterJournalSavedInstances.lua``):
```LUA
hooksecurefunc("EncounterJournal_ListInstances", UpdateFrames)
EncounterJournal_ListInstances()
```
By this:
```LUA
hooksecurefunc("EncounterJournal_ListInstances", UpdateFrames)
UpdateFrames()
```
Instead and report me if your issue is still occuring?
Based on your report, it happens when ``PLAYER_LOGIN`` event is triggered.

**Comment by benhamlin** [comment]
Yep, that fixes it!

**Comment by Meivyn** [comment]
Fixed in e27a81c1b7fa85c4a6b374c5c5823e72b8eaa5ef. Thanks for your report.
